report/report-core/src/test/java/com/mastercard/test/flow/report/seq/Browser.java

```diff
@@ -159,7 +161,15 @@ public WebDriver get() {
 				System.setProperty( CHROME_DRIVER_SILENT_OUTPUT_PROPERTY, "true" );
 				Logger.getLogger( "org.openqa.selenium" ).setLevel( Level.OFF );
 
-				return new ChromeDriver( options );
+				ChromeDriver driver = new ChromeDriver( options );
+
+				// Use Chrome DevTools to set the timezone
```

Comment:
I'm keenly aware that I write comments like this all the time myself, but we could improve this. Right now it's stating _what_ the below code does, but that code isn't particularly mysterious - we can already see that we're setting the timezone.
It'd be more valuable to describe _why_ we're doing that. Maybe we could add something like "Test expectations assume UTC"?

This is by no means a requirement for merge.